### DISCUSSION TITLE
Qdrant Server on Andriod

### DISCUSSION THREAD

**Discussion by harshakhmk** [qdrant/qdrant#3685] [General]
Hi, I am wondering if there is possible to run Qdrant Server in Andriod
If yes, could you kindly mention the steps required

**Comment by generall** [comment]
we have never tried

**Reply by harshakhmk** [reply]
Referencing this issue https://github.com/qdrant/qdrant/issues/2603, you mentioned executable is possible for Qdrant to run on Andriod using one package " upload rust binary action"
Just a confirmation, so Qdrant can be packaged as a .exe file and run on Andriod?
